### PR TITLE
End of sprint commit.

### PR DESCRIPTION
**Pull request by JeanMeijer**

### CODE REVIEW

**Comment by JeanMeijer** [comment]
@bhichem In conjunction with pull request from the front end team.

File: Language_tryouts/wordExtract.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    if continuous_chunk:
+        named_entity = " ".join(current_chunk)
+        if named_entity not in continuous_chunk and named_entity != "":
+            continuous_chunk.append(named_entity)
+
+    return continuous_chunk
+
+#param: tweet json with info described before, can be extended for array of tweets
+def sendToES(tweet):
```
Comment (bhichem) [inline Language_tryouts/wordExtract.py:42]: use snake_case in function naming instead of camelCase

File: Language_tryouts/wordExtract.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+for subtree in chunks.subtrees():
+        if subtree.label() == "KEYWORD":
+            sorted_keys += subtree.leaves()
+
+#find frequency dist of kwywords and show 3 most common
+fdistKeys = FreqDist(sorted_keys)
+fdistKeys.plot(3, cumulative=False)
+
+#named entity recognition
+fdistNames = FreqDist(get_continuous_chunks(inputfile))
```
Comment (bhichem) [inline Language_tryouts/wordExtract.py:78]: use snake_case in variables too 
follow this:
module_name, package_name, ClassName, method_name, ExceptionName, function_name, GLOBAL_CONSTANT_NAME, global_var_name, instance_var_name, function_parameter_name, local_var_name.

File: Language_tryouts/wordExtract.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+fdistNames.plot(3, cumulative=False)
```
Comment (bhichem) [inline Language_tryouts/wordExtract.py:79]: Better use Classes and functions instead of plain python code

**Review by bhichem — CHANGES_REQUESTED** [review]
Please process the comments by changing the code, or explaining why changes should not be done before merging.

**Review by bhichem — APPROVED** [review]

**Pull request merged**